yarn-project/aztec.js/src/example_contracts/easy_zk_token/artifacts/easy_zk_token.ts

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable */
```

Comment:
can these be symlinked instead, this could be a footgun when stuff changes underfoot